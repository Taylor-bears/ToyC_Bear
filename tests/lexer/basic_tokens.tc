int main() {
  if (x > 10) { return 0; }
}