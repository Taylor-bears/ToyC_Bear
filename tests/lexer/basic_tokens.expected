INT
ID(main)
LPAREN
RPAREN
LBRACE
IF
LPAREN
ID(x)
GT
NUMBER(10)
RPAREN
LBRACE
RETURN
NUMBER(0)
SEMICOLON
RBRACE
RBRACE
EOF